com.unity.render-pipelines.high-definition/Runtime/Sky/HDRISky/HDRISky.cs

```diff
@@ -150,8 +150,8 @@ public override int GetHashCode()
                 hash = hash * 23 + dirLightShadow.overrideState.GetHashCode();
                 hash = hash * 23 + rectLightShadow.overrideState.GetHashCode();
 #else
-                hash = hdriSky.value != null ? hash * 23 + hdriSky.GetHashCode() : hash;
-                hash = flowmap.value != null ? hash * 23 + flowmap.GetHashCode() : hash;
+                hash = hdriSky.value != null ? hash * 23 + CoreUtils.GetTextureHash(hdriSky.value) : hash;
```

Comment:
i think the code should be
`hash = hash * 23 + hdriSky.GetHashCode();`
otherwise `overrideState` is not taken into account by the hash

Also i'd say GetHashCode should be overriden in Texture2DParameter and co to use the correct hash function (CoreUtils.GetTextureHash), as i believe other systems relying on the hashcode can get wrong results because of that

Reply:
Yeah, that + all the other volume texture properties